cmd/crowdsec/metrics.go

```diff
@@ -78,6 +78,18 @@ func dumpMetrics() {
 }
 
 func registerPrometheus(config *csconfig.PrometheusCfg) {
+	if !config.Enabled {
+		return
+	}
+	if config.ListenAddr == "" {
+		log.Errorf("prometheus is enabled, but the listen address is empty, using '127.0.0.1'")
+		config.ListenAddr = "127.0.0.1"
+	}
+	if config.ListenPort == 0 {
+		log.Errorf("prometheus is enabled, but the listen port is empty, using '9090'")
```

Comment:
make it info I guess + it should be `6060` as it's the port used by `cscli` by default